test/functional/feature_assumeutxo.py

```diff
@@ -278,6 +275,18 @@ def check_tx_counts(final: bool) -> None:
 
         assert_equal(n1.getblockchaininfo()["blocks"], SNAPSHOT_BASE_HEIGHT)
 
+        self.log.info(f"Check that restarting with -reindex will delete the snapshot chainstate")
+        self.restart_node(1, extra_args=['-reindex=1', *self.extra_args[1]])
```

Comment:
This only covers `-reindex` but not `-reindex-chainstate`. It's easily possible when using node 2 for the test which is not pruned: https://github.com/fjahr/bitcoin/commit/38c5d80f73c7058f17ba8e89a3802ef981080864

Reply:
Thanks! I took your suggestion (with a minor change to logging).